app/src/main/res/values/strings.xml

```diff
@@ -73,13 +73,13 @@
     <string name="error_failed_to_launch_link">Could not open link.</string>
     <string name="error_filename_required">A filename is required.</string>
     <string name="error_file_not_create">Could not create file:</string>
-    <string name="error_invalid_db">Invalid database or unrecognized master key.</string>
+    <string name="error_invalid_db">Invalid database or unrecognized master keyfile.</string>
```

Comment:
The master key designates the password and key file (and potentially other means of authentication required) not just the key file